.vscode/launch.json

```diff
@@ -17,6 +17,17 @@
             "restart": true,
             "localRoot": "${workspaceFolder}",
             "remoteRoot": "."
-        }
+        },
+        {
+            "type": "pwa-node",
+            "request": "launch",
+            "name": "Debug Node.js",
+            "runtimeExecutable": "${workspaceFolder}/node_modules/.bin/ts-node-dev",
+            "args": ["--respawn", "--transpileOnly", "src/index.ts"],
+            "protocol": "inspector",
+            "console": "integratedTerminal",
+            "internalConsoleOptions": "neverOpen",
+            "skipFiles": ["<node_internals>/**"]
+          }
```

Comment:
Can we keep this here but drop the one above (named "Attatch to Server"). The other one is used by starting the server and then connecting to it - your one is easier to use.